lib/phoenix/html/tag.ex

```diff
@@ -0,0 +1,85 @@
+defmodule Phoenix.HTML.Tag do
+  @moduledoc ~S"""
+  Helpers related to producing HTML tags within templates.
+  """
+
+  alias Phoenix.HTML.Safe
+
+  @data_attrs [:method, :remote, :confirm]
+
+  @doc ~S"""
+  Creates an HTML tag with the given name and options.
+
+  ## Examples
+      iex> Phoenix.HTML.Tag.tag(:br)
```

Comment:
You can just have `tag(:br)` in the examples. I think that's how most people will use it anyway as the module will be automatically imported in views. When doing doctests, you just need to do `doctest Phoenix.HTML.Tag, import: true`.